test/JDBC/expected/sys-assembly_types.out

```diff
@@ -0,0 +1,6 @@
+SELECT * FROM sys.assembly_types
```

Comment:
This should have been part of the sys-assembly_types test in the input folder. But as discussed from other PRs, this is not needed since we already have the prepare/verify/cleanup format so I will remove this along with the test to avoid redundant tests.